roxie/ccd/ccdserver.cpp

```diff
@@ -1021,7 +1019,7 @@ extern IEngineRowStream *connectSingleStream(IRoxieAgentContext *ctx, IFinalRoxi
     return result;
 }
 
-class CRoxieServerActivity : implements CInterfaceOf<IRoxieServerActivity>, implements IFinalRoxieInput, implements IEngineRowStream, implements IRoxieContextLogger
+class CRoxieServerActivity : public CRoxieContextLogger, implements IRoxieServerActivity, implements IFinalRoxieInput, implements IEngineRowStream
```

Comment:
Need to think about this change.  By reordering the base interfaces it means that virtual calls into IRoxieServerActivity may be *slightly* less efficient.  I suspect it is unlikely to cause any issues.

Reply:
I think it is ok.  We might find moving IEngineRowStream has a minor performance impact - but not for this PR